.circleci/config.yml

```diff
@@ -56,6 +56,8 @@ jobs:
           key: v1-mvn-deps-{{ checksum "pom.xml" }}
       - restore_cache:
           key: v1-npm-deps-{{ checksum "package-lock.json" }}
+      - run: echo "$GPG_PRIVATE_KEY" | base64 --decode | gpg --import
+      - run: echo "default-key 7701193A898A849383D3E8B49F8AFEACBF07F7C4" > ~/.gnupg/gpg.conf
```

Comment:
I think the key fingerprint needs to be saved as an ENV var, but correct me if I am wrong.

Reply:
This is the ID of the key. The private key is stored as an env var in ci